lib/modules/platform/bitbucket/index.ts

```diff
@@ -303,12 +288,13 @@ export async function findPr({
   state = 'all',
 }: FindPRConfig): Promise<Pr | null> {
   logger.debug(`findPr(${branchName}, ${prTitle}, ${state})`);
-  const prList = await getPrList();
-  const pr = prList.find(
-    (p) =>
-      p.sourceBranch === branchName &&
-      (!prTitle || p.title.toUpperCase() === prTitle.toUpperCase()) &&
-      matchesState(p.state, state),
+  const pr = await getPrList().then((prs) =>
+    prs.find(
+      (p) =>
+        p.sourceBranch === branchName &&
+        (!prTitle || p.title.toUpperCase() === prTitle.toUpperCase()) &&
+        matchesState(p.state, state),
+    ),
```

Comment:
why this change?

Reply:
I don't know, intention just feels cleaner to me

Reply:
but with this change the code is more nested that before 🤔

I prefer await than then, especially when the callback function is more than two loc